CHANGELOG.md

```diff
@@ -1,9 +1,16 @@
 OhMyThreads.jl Changelog
 =========================
 
+Version 0.7.0
+-------------
+- ![BREAKING][badge-breaking] We now use ChunkSplitters version 3.0. The function `OhMyThreads.chunks` has been renamed to `OhMyThreads.index_chunks`. The new functions `index_chunks` and `chunks` (different from the old one with the same name!) are now exported. See ChunkSplitters.jl for more information.
```

Comment:
oh I had missed that we opted for `chunks` instead of `chunk`. I like the name `chunks` more than `chunk`, though I am a bit concerned about this causing some confusing bugs. Will probably be fine though (and not really a concern for this repo anyways)